api/core/helpers.py

```diff
@@ -14,21 +14,26 @@ def send_login_email(user, request, *, welcome):
     token = get_query_string(user)
     url = base + token
 
-    # TODO: Convert this to an email template
-    if welcome:
-        subject = "Welcome to Voter Engagement"
-    else:
-        subject = "Greetings from Voter Engagement"
-    body = f"Click here to log in: {url}"
-    email = EmailMessage(
-        subject=subject,
-        body=body,
+    message = EmailMessage(
+        subject=None,
         from_email="Citizen Labs <noreply@citizenlabs.org>",
         to=[user.email],
     )
-
-    log.debug(f"Sending email: {prettify(email.__dict__)}")
-    count = email.send(fail_silently=False)
+    if welcome:
+        message.template_id = 'voter-engagement-welcome'
+    else:
+        message.template_id = 'voter-engagement-login'
+    message.merge_global_data = {
+        'FIRST_NAME': user.first_name,
+        'LAST_NAME': user.last_name,
+        'LOGIN_URL': url,
+        # TODO: Implement unsubscribe functionality
+        'UNSUBSCRIBE_URL': 'https://citizenlabs.org/contact/',
+        'ABOUT_URL': 'https://citizenlabs.org/about/',
+    }
```

Comment:
@dianeumoh The above variables will be available in email templates in the form `*|FIRST_NAME|*`.